specification/agrifood/resource-manager/readme.java.md

```diff
@@ -0,0 +1,12 @@
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
+
+``` yaml $(java)
+java:
+  fluent: true
+  namespace: com.azure.agrifood
+  license-header: MICROSOFT_MIT_NO_CODEGEN
+  output-folder: $(azure-libraries-for-java-folder)/agrifood/arm-agrifood
```

Comment:
You don't need any of them.

But where is the `title: AgriFoodMgmtClient`?

Reply:
updated.

Reply:
Use `AgriFood` as prefix for entry of Java lib.

Use `AgriFoodManagementClient` (this is usually not visible to user) as Java does not prefer (inappropriately) abbreviated word ("Mgmt").